src/api/localResolvers/experiment.js

```diff
@@ -34,6 +34,39 @@ function serializeExpCookie(assignments) {
 	return filteredStrings.join('|');
 }
 
+/**
+ * Cycle through targets object and determine matches
+ *
+ * @param {object} targets
+ * @param {object} cookieStore
+ * @returns {boolean}
+ */
+function matchTargets(targets, cookieStore) {
+	// return true if no targets are set, aka everyone matches!!!
+	if (_isUndefined(targets)) return true;
+
+	// re-start if targets are present
+	let matched = false;
+
+	// User Segment Targets
```

Comment:
I can simplify some of these nested ifs...and we'll likely want to separate the 'users' checks at some point...
- targets will be defined if we get here...so just check for targets.users
- should set a var for cookieStore.get('kvu')
- combine kvuValue with check for 'cookied' or 'uncookied'

Reply:
Did thEse things...